client/src/App.js

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
-
-import SavedList from './Movies/SavedList';
-import MovieList from './Movies/MovieList';
-import Movie from './Movies/Movie';
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
You are importing Link and never using it, not a big deal, but it's good practice to clean up your code before pushing it. 
When your code is compiled there are 2 warnings you could want to get rid of. This is the first.